bin/workflows/pycbc_create_offline_search_workflow

```diff
@@ -426,6 +426,13 @@ table = wf.make_foreground_table(workflow, combined_bg_file,
                                  singles=insps, extension='.html',
                                  tags=combined_bg_file.tags)
 
+fore_xmlall = wf.make_foreground_table(workflow, combined_bg_file,
+                    hdfbank, rdir['open_box_result'], singles=insps,
+                    extension='.xml', tags=["xmlall"])
+fore_xmlloudest = wf.make_foreground_table(workflow, combined_bg_file,
+                    hdfbank, rdir['open_box_result'], singles=insps,
+                    extension='.xml', tags=["xmlloudest"])
```

Comment:
Looks good. Just make sure that any relevant config files contain the options related to these tags!